### ISSUE TITLE
Failing tests

### ISSUE THREAD

**Issue by jenssegers** [jenssegers/chef-patch#19]
Hey @ramonskie @mvdkleijn,

I'm having some issues with the current tests after I did some changes. I basically removed the `converge_by` and `ruby_block`: https://github.com/jenssegers/chef-patch/compare/v2.2.0...master

Could any of you point me in the right direction to fix the tests?

**Comment by ramonskie** [comment]
first thing i noticed you are using tabs in the resources files.
but this does not causes your problems

i would also like to know what the exact issue is
because i get the following error

```
Errno::ENOENT
-------------
No such file or directory @ rb_sysopen - /test1

Cookbook Trace:
---------------
/tmp/d20160229-17209-k74o32/cookbooks/patch/resources/append_line.rb:19:in `block in class_from_file'

Resource Declaration:
---------------------
# In /tmp/d20160229-17209-k74o32/cookbooks/test/recipes/append.rb

  1: append_line '/test1' do
  2:   line 'test'
  3: end
  4: 
```

**Comment by jenssegers** [comment]
Yeah that's the same one: https://travis-ci.org/jenssegers/chef-patch/jobs/112261745

I don't really know why because the spec says:

```
expect(File).to receive(:read).with("/test1").and_return('')
```

I'm quite the Ruby noob :)

**Comment by jenssegers** [comment]
Or is it because of:

```
before = Digest::SHA256.file(file_path).hexdiges
```

**Comment by ramonskie** [comment]
sorry i have no clue at the moment
i will ask around if somebody can help.

btw you removed the following line https://github.com/jenssegers/chef-patch/commit/1371f614daab3169538beb509564c9e542dacd12

can i ask why?
because just a hash calculation is not going to do it here i think
lets say if you do 2 alterations in in that file
then the file is changed 2 times

**Comment by jenssegers** [comment]
The hash is only used to check if the file changed after running the FileEdit method and trigger notifications. Could you give an example in which case this would fail?

**Comment by ramonskie** [comment]
lets first fix your test issue
before changing this subject :+1:

**Comment by mvdkleijn** [comment]
I took a quick look and can only say that the system is trying to really find a file /test1 on disk rather than using the mocked setup from the test. It doesn't work for me locally either so I'll have to dig a little deeper.

**Comment by mvdkleijn** [comment]
Sorry about this one... I really haven't had any chance to work on it more.
